app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
-<p>Here is a details of a given user</p>
\ No newline at end of file
+<p>Here is a details of a given user</p>
+
+<div class ='user'>
+  <img class="user_img" src=<%= @user.photo %> alt=<%= @user.name %> />
+  <%= render "share/user",   user: @user  %>
+</div>
+
+<div class = details>
+  <div class = 'bio'>
+    <h2>Bio</h2>
+    <p><%= @user.bio  %></p>
+  </div>
+
+   <% @user.posts.each do |post, index| %>
+     <div class='user_posts'>
+      <%= render "share/posts",   post: post %>
+    </div>
+   <% end %>
```

Comment:
- [ ] It will be great to use the recent post method you have created in your user's model to return the three most recent posts. Here in your implementation, you are returning all the posts created by the user but the requirement is to display the three most recent posts.